pkgs/applications/networking/pjsip/default.nix

```diff
@@ -4,8 +4,11 @@
 , fetchpatch
 , openssl
 , libsamplerate
+, python
```

Comment:
The plain `pjsip` library won't build now because this is a mandatory argument.

Reply:
Come to think, if you make this `python3` (python 2 is dead) and enable python support by default, `pkgs.pjsip` and `python3Packages.pjsua2` will share the same derivation. `pkgs.pjsip` _shouldn't_ be bloated by the python bindings as that should all end up in the `py` output which will be ignored by packages that don't need it.

Reply:
I've changed it back to using `python3` and enable python support by default :+1: